optuna/integration/lightgbm.py

```diff
@@ -62,60 +63,85 @@ class LightGBMPruningCallback(object):
             Note that this argument will be ignored if you are calling
             `cv method <https://lightgbm.readthedocs.io/en/latest/Python-API.html#lightgbm.cv>`_
             instead of train method.
+        report_interval:
+            Check if the trial should be reported for pruning every n-th epoch.
+            By default ``report_interval=1`` and reporting is performed after every epoch.
+            Note that the pruning itself is performed according to the interval definition
+            of the pruner.
```

Comment:
- I guess a trial reports the intermediate values, but the first sentence says that the trial is reported.
-  "boosting iteration" or "boosting round" is more suitable for lightgbm than "epoch".
https://lightgbm.readthedocs.io/en/latest/pythonapi/lightgbm.train.html#lightgbm-train

```suggestion
        report_interval:
            Check if the trial should report intermediate values for pruning every n-th boosting iteration.
            By default ``report_interval=1`` and reporting is performed after every iteration.
            Note that the pruning itself is performed according to the interval definition
            of the pruner.
```